caravel/assets/webpack.config.js

```diff
@@ -7,6 +7,11 @@ const APP_DIR = path.resolve(__dirname, './');
 // output dir
 const BUILD_DIR = path.resolve(__dirname, './dist');
 
+const fs = require('fs');
+const contents = fs.readFileSync('package.json');
+const JSONObject = JSON.parse(contents);
+const VERSION_STRING = JSONObject.version;
```

Comment:
could shorten these lines to since we won't need the `contents` or `JSONObject` vars later:
`const VERSION_STRING = JSON.parse((fs.readFileSync('package.json')).version;`